src/components/Link/SbLink.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <component
+    :is="as"
+    class="sb-link"
+    :class="classes"
+    :title="title"
+    :href="href"
+    :to="to"
+    v-on="$listeners"
+  >
+    <SbIcon v-if="icon" :size="'normal'" :name="icon"></SbIcon>
```

Comment:
I think the `:size` can be removed

Reply:
please, use self-close for components without content: https://v2.vuejs.org/v2/style-guide/?redirect=true#Self-closing-components-strongly-recommended

Reply:
@gustavomelki  i've self-closed the components without content, please review. Thanks